WorkoutGeneratorApp/package.json

```diff
@@ -7,13 +7,15 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "freshtall": "npm cache clean --force && rmdir /s node_modules && del -f package-lock.json && npm install"
```

Comment:
What is `freshtall`? I'm guessing it had to do with your issues of trying to run it? Or is this just for convenience?

Reply:
This is not an issue I had per say. Any one can run these commands to get a fresh start up, that's why I thought it'd be nice to include in the package. Now all you need to do is call a single command and packaging conflicts at pull get solved pretty easily (if need be)